test/import/import-cli-config-sync.test.ts

```diff
@@ -0,0 +1,95 @@
+  const tempDir = fs.mkdtempSync(path.join(os.tmpdir(), 'cdk8s-import-test'));
+  const fixture = path.join(tempDir, 'fixture.yaml');
+  try {
+    if (Array.isArray(data)) {
+      fs.writeFileSync(fixture, data.map(d => yaml.stringify(d)).join('\n---\n'));
+    } else {
+      fs.writeFileSync(fixture, yaml.stringify(data));
+    }
+    await test(fixture, configFilePath, tempDir);
+  } finally {
+    fs.removeSync(tempDir);
+    // clean up cdk8s.yaml file, reseting back to template:
+    const defaultConfig = yaml.parse(fs.readFileSync(path.join(configFilePath, 'cdk8s-template.yaml'), 'utf-8'));
+    await fs.outputFile(path.join(configFilePath, 'cdk8s.yaml'), yaml.stringify(defaultConfig));
+  }
+}
+
+test('reading default config file works', async () => {
+
+  // clean up cdk8s.yaml file, reseting back to template:
+  const defaultConfig = yaml.parse(fs.readFileSync(path.join(configFilePath, 'cdk8s-template.yaml'), 'utf-8'));
+  await fs.outputFile(path.join(configFilePath, 'cdk8s.yaml'), yaml.stringify(defaultConfig));
+  const config1 = readConfigSync(configFilePath);
+  expect(config1.language).toEqual('typescript');
+  expect(config1.app).toEqual('node main.js');
+  expect(config1.imports?.length == 1).toBeTruthy();
+  expect(config1.imports?.includes('k8s')).toBeTruthy();
+});
+
+test('writing default config file works', async () => {
+  await addImportToConfig(jenkinsCRD, configFilePath);
+  const config2 = readConfigSync(configFilePath);
+  expect(config2.imports).toEqual(['k8s', jenkinsCRD]);
+  expect(config2.language).toEqual('typescript');
+  expect(config2.app).toEqual('node main.js');
+
+  // test importing duplicate does not add to config
+  await addImportToConfig(jenkinsCRD, configFilePath);
+  const config3 = readConfigSync(configFilePath);
+  expect(config3).toEqual(config2);
+
+  // clean up cdk8s.yaml file, reseting back to template:
+  const defaultConfig = yaml.parse(fs.readFileSync(path.join(configFilePath, 'cdk8s-template.yaml'), 'utf-8'));
+  await fs.outputFile(path.join(configFilePath, 'cdk8s.yaml'), yaml.stringify(defaultConfig));
+});
+
+test('can import crds from fixtures', async () => {
+
+  const crd: ManifestObjectDefinition = {
+    apiVersion: 'apiextensions.k8s.io/v1beta1',
+    kind: 'CustomResourceDefinition',
+    metadata: {
+      name: 'testMetadata',
+    },
+    spec: {
+      version: 'v1',
+      group: 'testGroup',
+      names: {
+        kind: 'testNameKind',
+      },
+      validation: {
+        openAPIV3Schema: {
+          description: "*/console.log('hello')/*",
+        },
+      },
+    },
+  };
+
+  await withTempFixture(crd, async (fixture: string, configFile: string, cwd: string) => {
+    const config1 = readConfigSync(configFile);
+    expect(fixture).not.toEqual(configFile);
+    expect(config1.language).toEqual('typescript');
+    expect(config1.app).toEqual('node main.js');
+    expect(config1.imports).toEqual(['k8s']);
```

Comment:
What are you asserting on here? that the `readConfigSync` function reads the file correctly? We already have tests for that...no need to re-assert.